package.json

```diff
@@ -56,16 +56,16 @@
     "i18next": "^19.8.3",
     "i18next-browser-languagedetector": "^6.0.1",
     "lodash.merge": "^4.6.2",
-    "mini-css-extract-plugin": "^1.2.1",
-    "optimize-css-assets-webpack-plugin": "^5.0.4",
-    "react": "^16.13.1",
+    "lodash.mergewith": "^4.6.2",
```

Comment:
I think @pmuir is bringing in another lodash library, wonder if it's worth bringing in full lodash and making more specific requires?

Reply:
Possibly! That should come as a dedicated refactor in a future PR, worth another issue.